### DISCUSSION TITLE
What is difference between VRMRig and VRMHumanoidRig? (RawBoneNode vs NormalizedBoneNode)

### DISCUSSION THREAD

**Discussion by seastar105** [pixiv/three-vrm#1085] [Help] [answered]
As mentioned in title, what is difference between normalized bone and raw bone?? 

It seems `getBoneNode` has been deprecated since `NormalizedBoneNode` added. What's difference?

**Comment by 0b5vr** [comment, marked as answer]
Would the "Normalized Human Bones" section of the Migration Guide helps you understand?

https://github.com/pixiv/three-vrm/blob/dev/docs/migration-guide-1.0.md#normalized-human-bones

Also, `VRMRig` and `VRMHumanoidRig` are not intended to be touched by end developers, so don't worry about it.

**Reply by seastar105** [reply]
Thanks @0b5vr. I got it.
